setup-stage2.sh

```diff
@@ -26,13 +26,15 @@ echo "Making sure that Wine isn't running anything..."
 # Close all Wine ".exe" processes.
 # NOTE: Since we scan for the newest PID each time, we'll properly close
 # the programs in the reverse order of how they were started!
-# TODO/FIXME: This is harmful if there are other Wine runtimes on the system
+# T ODO/F IXME: This is harmful if there are other Wine runtimes on the system
```

Comment:
Seems like an accidental change here?

Reply:
That's actually a good reminder.

I was hoping to discuss the possibility of replacing that block with a `WINEPREFIX=$HOME/.xlcore/wineprefix wineserver -k;` since that's the best way to kill all wine processes running inside a given prefix.

Reply:
That'll need tested, I've run into issues where that won't kill everything but still exit with `$?=0`, or will hang.

Reply:
Understood, will leave unchanged for now and revert the todo